src/pages/add/copayers/copayers.ts

```diff
@@ -0,0 +1,113 @@
+import { Component } from '@angular/core';
+import { NavController, NavParams, Events } from 'ionic-angular';
+import { Logger } from '@nsalaun/ng-logger';
+
+// Pages
+import { HomePage } from '../../../pages/home/home';
+import { WalletDetailsPage } from '../../../pages/wallet-details/wallet-details';
+
+// Providers
+import { AppProvider } from '../../../providers/app/app';
+import { BwcErrorProvider } from '../../../providers/bwc-error/bwc-error';
+import { OnGoingProcessProvider } from "../../../providers/on-going-process/on-going-process";
+import { PlatformProvider } from '../../../providers/platform/platform';
+import { PopupProvider } from '../../../providers/popup/popup';
+import { ProfileProvider } from '../../../providers/profile/profile';
+import { WalletProvider } from '../../../providers/wallet/wallet';
+
+
+@Component({
+  selector: 'page-copayers',
+  templateUrl: 'copayers.html',
+})
+export class CopayersPage {
+
+  public appName: string = this.appProvider.info.userVisibleName;
+  public appUrl: string = this.appProvider.info.url;
+  public isCordova: boolean = this.platformProvider.isCordova;
+
+  public wallet: any;
+  public shareIcon: string;
+  public copayers: any;
+  public secret: any;
+
+  constructor(
+    private appProvider: AppProvider,
+    private bwcErrorProvider: BwcErrorProvider,
+    private events: Events,
+    private logger: Logger,
+    private navCtrl: NavController,
+    private navParams: NavParams,
+    private platformProvider: PlatformProvider,
+    private popupProvider: PopupProvider,
+    private profileProvider: ProfileProvider,
+    private onGoingProcessProvider: OnGoingProcessProvider,
+    private walletProvider: WalletProvider
+  ) {
+
+  }
+
+  ionViewDidLoad() {
+    console.log('ionViewDidLoad CopayersPage');
+    this.wallet = this.profileProvider.getWallet(this.navParams.data.walletId);
+    this.updateWallet();
+    this.shareIcon = this.platformProvider.isIOS ? 'iOS' : 'Android';
```

Comment:
Is it used?
We should use one, not depending on the platform.

Reply:
It isn't used at the moment. I will remove it